src/main/java/cart/controller/ProductController.java

```diff
@@ -0,0 +1,33 @@
+package cart.controller;
+
+import java.util.List;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import cart.controller.response.ProductResponse;
+import cart.service.GeneralProductService;
+
+@Controller
+public class ProductController {
+	private final GeneralProductService generalProductService;
```

Comment:
여기서는 서비스의 인터페이스가 아닌 구현체를 사용하신 이유가 있을까요~?

Reply:
앗 이유가 없습니다 ㅠㅠ 다형성을 이용할 수도 있도록 수정했습니다 !
```suggestion
	private final ProductService productService;
```

Reply:
네넵~~ 수정된 내용이 푸시되지 않은 것 같아서 요건 다음 미션에서 확인 부탁해요~!